transforms/language/extreme_tokenized/dpk_extreme_tokenized/runtime.py

```diff
@@ -0,0 +1,123 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import sys
+from argparse import ArgumentParser, Namespace
+
+from data_processing.data_access import DataAccessFactory
+from data_processing.runtime.pure_python import (
+    PythonTransformLauncher,
+    PythonTransformRuntimeConfiguration,
+)
+from data_processing.transform import TransformConfiguration
+from data_processing.utils import CLIArgumentProvider, ParamsUtils
+from dpk_extreme_tokenized.common import (
+    arrow_path_cli_param,
+    arrow_path_default,
+    cli_prefix,
+    contents_column_name_cli_param,
+    contents_column_name_default,
+    extreme_tokenized_data_access_key,
+    extreme_tokenized_data_factory_key,
+    logger,
+    short_name,
+)
+from dpk_extreme_tokenized.transform import ExtremeTokenizedTransform
+
+
+class ExtremeTokenizedTransformConfiguration(TransformConfiguration):
+    """
+    Provides support for configuring and using the associated Transform class include
+    configuration with CLI args and combining of metadata.
+    """
+
+    def __init__(self):
+        super().__init__(
+            name=short_name,
+            transform_class=ExtremeTokenizedTransform,
+            remove_from_metadata=[extreme_tokenized_data_access_key, extreme_tokenized_data_factory_key],
+        )
+        self.daf = None
+
+    def add_input_params(self, parser: ArgumentParser) -> None:
+        """
+        Add Transform-specific arguments to the given parser.
+        This will be included in a dictionary used to initialize the BlockListTransform.
+        By convention a common prefix should be used for all mutator-specific CLI args
+        (e.g, noop_, pii_, etc.)
+        """
+
+        parser.add_argument(
+            f"--{contents_column_name_cli_param}",
+            type=str,
+            required=False,
+            default=contents_column_name_default,
+            help="Name of the column holding the document text",
+        )
+        parser.add_argument(
+            f"--{arrow_path_cli_param}",
+            type=str,
+            required=True,
+            default=arrow_path_default,
+            help=" Arrow folder location.",
+        )
+        self.daf = DataAccessFactory(cli_prefix, False)
+        # Add the DataAccessFactory parameters to the transform's configuration parameters.
+        self.daf.add_input_params(parser)
+        self.params[extreme_tokenized_data_access_key] = self.daf.create_data_access()
+
+    def apply_input_params(self, args: Namespace) -> bool:
+        """
+        Validate and apply the arguments that have been parsed
+        :param args: user defined arguments.
+        :return: True, if validate pass or False otherwise
+        """
+        # Capture the args that are specific to this transform
+        captured = CLIArgumentProvider.capture_parameters(args, cli_prefix, False)
```

Comment:
Suggest you use cliprefix_<parameters> throughout your code. For that to work, you need to change from False to True 
```suggestion
        captured = CLIArgumentProvider.capture_parameters(args, cli_prefix, True)
```

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/965/commits/8cd8d5dddd1e11583087fe892cefaa681ffd067f